iothub/device/src/Authentication/ClientAuthenticationWithSakRefresh.cs

```diff
@@ -7,13 +7,33 @@
 
 namespace Microsoft.Azure.Devices.Client
 {
-    // Implementing SAS Token refresh based on a SharedAccessKey (SAK).
-    internal class ClientAuthenticationWithSakRefresh : ClientAuthenticationWithTokenRefresh
+    /// <summary>
+    /// Authentication method that implements SAS Token refresh based on a SharedAccessKey (SAK).
+    /// </summary>
+    public class ClientAuthenticationWithSakRefresh : ClientAuthenticationWithTokenRefresh
```

Comment:
nit: we should expand "sak" to shared access key, since not everyone would be familiar with that acronym